src/EventSourcingDb/Client.cs

```diff
@@ -500,23 +525,30 @@ public async IAsyncEnumerable<JsonElement> RunEventQlQueryAsync(
         }
     }
 
-    public async IAsyncEnumerable<TRow> RunEventQlQueryAsync<TRow>(
-        string query,
-        [EnumeratorCancellation] CancellationToken token = default)
+    private TRow DeserializeRow<TRow>(JsonElement payload)
     {
-        await foreach (var jsonElement in RunEventQlQueryAsync(query, token))
+        TRow? row;
+        try
         {
-            TRow? row;
-            try
+            if (typeof(TRow) == typeof(Event))
             {
-                row = jsonElement.Deserialize<TRow>(_defaultSerializerOptions);
+                var cloudEvent = payload.Deserialize<CloudEvent>(_defaultSerializerOptions);
+                if (cloudEvent == null)
```

Comment:
Nur aus Neugier – was ist in C# der Unterschied zwischen `== null` und `is null`? Manchmal sehe ich das eine, manchmal das andere.

Reply:
Ah danke, das war mir nicht klar, aber das ergibt absolut Sinn. Dann lass uns das gerne mit `is null` machen.